README.md

```diff
@@ -42,12 +44,12 @@ nosetests core/test_parameters.py:test_parameter_for_period
 
 ## Serving the API
 
-OpenFisca-Core provides a Web-API. It is served on the `6000` port.
+OpenFisca-Core provides a Web-API. It is by default served on the `5000` port.
 
-To run it with the mock country package `openfisca_country_template` and another port value as `5000`, run:
+To run it with the mock country package `openfisca_country_template` and another port value such as `2000`, run:
```

Comment:
I feel we should mention that `openfisca_country_template` should be installed with the api extra requirements.
> proposition : 

To serve the Web-API, you need to install a country package with the [web_api] extra require.
e.g.
```
pip install openfisca-country-template[web_api]
```
To run it with the mock country package `openfisca_country_template` and another port value such as `2000`, run:
[...]

Reply:
Actually, in the country template [PR](https://github.com/openfisca/country-template/pull/51/files) there is no `web_api` extra dependencies and it is required by default. 

Also it is [not possible](https://github.com/openfisca/openfisca-core/pull/703#issuecomment-409800029) to install the web api with something like `pip install Openfisca-France[web-api]` that would require `Openfisca-Core[web-api]`.